### DISCUSSION TITLE
Wrong info in wiki: Dark reader only responds to the system mode, not app mode, as the wiki says

### DISCUSSION THREAD

**Discussion by FeralFlora** [AutoDarkMode/Windows-Auto-Night-Mode#825] [General]
In the [wiki page on Dark Reader](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/Dark-Mode-for-Webbrowser), it says:

> As soon as Auto Dark Mode changes your Windows App Theme, Dark Reader will react to that displaying all pages in a dark look.

However, this is incorrect. Dark Reader only responds to System theme, not the app theme. So, if you're like me, and you have System theme set to always dark, and a light / dark schedule for the apps, Dark Reader will always be in dark mode. It won't respond to the app schedule.

This seems to be an imprecision, because elsewhere, the wiki says:

> And the best thing is: this extension can also react to the system theme!

So both system theme and app theme are mentioned interchangeably, but only the former is right, unfortunately.

See this enhancement request for Dark Reader: https://github.com/darkreader/darkreader/issues/11614

**Comment by Armin2208** [comment]
My Dark Reader on both edge and firefox respons to the app theme rather to the system theme. So I can't confirm your issue.

**Reply by FeralFlora** [reply]
Based on some tests (see demo below 👇), it seems that Dark Reader only follows the app theme *if* you set the Firefox theme (under `about:addons`) to "System theme - auto". Otherwise, Dark Reader in auto mode will actually follow the Firefox theme.

![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/33608083/323f3be4-51f2-4480-bcfc-5805bed742c1)

If you set the Firefox theme to dark mode (like I had), then Dark reader will not follow the system app theme, but will instead make the website dark, irrespective of the app or system theme. I assume that you've set the Firefox theme to "System theme"?

In summary, my configuration is like this:
- System theme: Always dark
- App theme: Light / dark schedule using Auto Dark Mode:
![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/33608083/eaedc5da-a730-4110-a222-db218653be09)
- Firefox theme: Dark
- Dark Reader: Auto

Result: Website are dark, even if app theme is light:

![firefox_2u8fLVeclr](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/33608083/0800db05-2b14-49a6-b6e2-45075086fb22)


With this configuration, it is apparent that Dark Reader in auto mode actually follows the Firefox theme, not the app theme. As both my system theme and Firefox theme stay dark, I mistakenly thought that it was following the system theme, but actually, it follows the Firefox theme.

**Reply by Armin2208** [reply]
I've tested the behavior of Google Chrome + Dark Reader, Firefox + Dark Reader and MS Edge + Dark Reader on a virtual machine. On Firefox and Edge the Dark Reader will respond to the browser theme. Both browsers respond to the App theme of Windows. On Chrome, it always responds to the Windows app theme regardless of Chrome's settings. 

Based on your notice and suggestion, I've updated the [wiki page](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/Dark-Mode-for-Webbrowser). New is the section explaining how to enable automatic theme on the browser itself. The wording changed from "System theme" to "App theme". And I updated the screenshots of the Dark Reader set up tutorial to respect the UI changes in the latest updates. 

Thank you!

**Reply by FeralFlora** [reply]
Nice, and you're welcome! Thanks for looking into it, glad we could clear that up. With the new instructions, users should be able to get it working as expected. Dark Reader should be clearer about this themselves.
